testing/tests/DevExpress.ui.widgets/sortable.tests.js

```diff
@@ -2615,6 +2618,8 @@ QUnit.module('With scroll', getModuleConfigForTestsWithScroll('#itemsWithScroll'
     });
 
     QUnit.test('Placeholder should be visible after page scrolling (T871213)', function(assert) {
+        const fixtureRoot = document.querySelector('#qunit-fixture');
```

Comment:
Why do we need this change?

Reply:
This test was the most difficult to adjust to shadow Dom. We need to modify the `#qunit-fixture` element itself, but not it's shadow container. This change is specific only for this test